noaa_gefs_aggregate.R

```diff
@@ -0,0 +1,17 @@
+vars <- c("tmax", "tmin", "prcp", "tdew", "wind", "srad", "tavg")
+
+funcs <- c("max", "min", "mean", "sum", "mean", "sum", "mean")
+
+for (x in seq_len(length(data))) {
```

Comment:
Looks great @ajakucewicz . My mistake though. Let's remove the first loop that loops through the sites or the `data` list and turn the rest into a function to move on to the next step. 
Thank you